python/ray/_private/worker.py

```diff
@@ -2650,6 +2662,15 @@ def get(
         if isinstance(object_refs, CompiledDAGRef):
             return object_refs.get(timeout=timeout)
 
+        if isinstance(object_refs, list):
+            allCompiledDAGRefs = True
+            for object_ref in object_refs:
+                allCompiledDAGRefs = allCompiledDAGRefs and isinstance(
+                    object_ref, CompiledDAGRef
+                )
+            if allCompiledDAGRefs:
+                return [object_ref.get(timeout=timeout) for object_ref in object_refs]
```

Comment:
I think we also need to handle the else condition. It's okay to throw an error for now if the list contains a CompiledDAGRef but not all CompiledDAGRefs.